CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.32.0] - 2022-03-14
```

Comment:
Do we need a major version bump for this? 
A minor version bump seems to be good enough to me.

Reply:
sure